components/invitation_modal/index.tsx

```diff
@@ -70,9 +70,9 @@ export function mapStateToProps(state: GlobalState, props: OwnProps) {
     });
     const guestAccountsEnabled = config.EnableGuestAccounts === 'true';
     const emailInvitationsEnabled = config.EnableEmailInvitations === 'true';
-    const isLicensed = license && license.IsLicensed === 'true';
+    const isEnterpriseReady = config.BuildEnterpriseReady === 'true';
     const isGroupConstrained = Boolean(currentTeam.group_constrained);
-    const canInviteGuests = !isGroupConstrained && isLicensed && guestAccountsEnabled && haveICurrentTeamPermission(state, Permissions.INVITE_GUEST);
+    const canInviteGuests = !isGroupConstrained && isEnterpriseReady && guestAccountsEnabled && haveICurrentTeamPermission(state, Permissions.INVITE_GUEST);
```

Comment:
I replaced this `isLicensed` check with `isEnterpriseReady` because we need the option to invite guests to be available for self hosted starter in order for the upsell to appear